nrf70_bm_lib/docs/source/nrf70_bm_porting_guide.rst

```diff
@@ -0,0 +1,65 @@
+.. _nrf70_bm_porting_guide:
+
+nRF70 BM Porting Guide
+======================
+The nRF70 BM library is designed to be portable across different platforms.
```

Comment:
Same comment as above. I suggest we use the term Library for the RTOS-agnostic part and Driver for the whole solution.